test/integration/config/env.js

```diff
@@ -0,0 +1,3 @@
+require('../../env');
+
+process.env.ENABLE_CONFIRMATION_REDESIGN="true"
```

Comment:
Should we be enabling this globally? Presumably we'd only want this for integration tests of the confirmation redesign

Reply:
That is old code that stayed behind. Redesigned signature pages are no longer behind that env variable. I am removing it from the test env.

Reply:
done 👍